package.json

```diff
@@ -22,7 +25,17 @@
     "eslint": "^8.38.0",
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-react-refresh": "^0.3.4",
+    "prettier": "^2.8.8",
+    "husky": "^8.0.0",
+    "lint-staged": "^13.2.2",
     "typescript": "^5.0.2",
     "vite": "^4.3.2"
+  },
+  "lint-staged": {
+    "src/**/*.{ts,scss,tsx,js,html,json,md}": [
+      "yarn run lint",
+      "yarn run lint-fix",
+      "yarn run format"
+    ]
```

Comment:
1. this is good but make sure to add command to test too! because husky will test before coming, so add test command here.
2 . And because you are waiting for the jest to be configured, after it is configured you can test this project.

Reply:
Yes